Dropdown/Dropdown.js

```diff
@@ -365,23 +366,35 @@ const DropDownExtended = hoc((config, Wrapped) => {
 
 			if (!prevProps.open && open) {
 				on('click', this.handleClick);
+				on('keydown', this.handleKeyDown);
 			} else if (prevProps.open && !open) {
 				off('click', this.handleClick);
+				off('keydown', this.handleKeyDown);
 			}
 		}
 
 		componentWillUnmount () {
 			off('click', this.handleClick);
+			off('keydown', this.handleKeyDown);
 		}
 
 		clickedOutsideDropdown = ({target}) => !this.node.contains(target);
 
+		escKeyPressed = (event) => {
+			return (event.keyCode === 27);
+		};
+
 		// If a click happened outside the component area close the dropdown by forwarding the onClick from Toggleable.
 		handleClick = handle(
 			this.clickedOutsideDropdown,
 			forward('onClick')
 		).bindAs(this, 'handleClick');
 
+		handleKeyDown = handle(
+			this.escKeyPressed,
```

Comment:
Let's `import {forKey} from '@enact/core/handle';` and replace this to `forKey('cancel')`.

Reply:
Also, we need to add
```
// Add keymap for escape key
add('cancel', 27);
```
above line 63 after importing `add` like `import {add} from '@enact/core/keymap';`